pages/_sites/[site]/tinycms/homepage.js

```diff
@@ -72,15 +72,25 @@ export default function HomePageEditor({
     let article2 = null;
     let article3 = null;
 
-    if (selectedLayout.name !== 'Big Featured Story' && subFeaturedTopArticle) {
+    if (selectedLayout.name === 'Large Package Story Lead') {
```

Comment:
It looks like we have duplicate if-statements here and lines 79:
```if (selectedLayout.name === 'Large Package Story Lead') {
// do something
}```

If the change to the condition on line 75 was intentional, we can combine them so it's cleaner. Like this:
```
if (selectedLayout.name === 'Large Package Story Lead') {
      article2 = subFeaturedTopArticle.id;
      article3 = subFeaturedBottomArticle.id;
}
```